qa/smoke-test-http/src/test/java/org/elasticsearch/http/snapshots/RestGetSnapshotsIT.java

```diff
@@ -176,7 +177,14 @@ public void testSortAndPaginateWithInProgress() throws Exception {
             inProgressSnapshots.add(AbstractSnapshotIntegTestCase.startFullSnapshot(logger, repoName, snapshotName, false));
         }
         AbstractSnapshotIntegTestCase.awaitNumberOfSnapshotsInProgress(logger, inProgressCount);
-
+        AbstractSnapshotIntegTestCase.awaitClusterState(
+            logger,
+            state -> state.custom(SnapshotsInProgress.TYPE, SnapshotsInProgress.EMPTY)
+                .asStream()
+                .flatMap(s -> s.shards().stream())
+                .filter(e -> e.getKey().getIndexName().equals("test-index-1"))
```

Comment:
`filter` and `allMatch` can be collapsed into just `allMatch`

Reply:
Yeah, I can do 

```
e -> e.getKey().getIndexName().equals("test-index-1") == false
                        && e.getValue().state() == SnapshotsInProgress.ShardState.SUCCESS
```

like in `GetSnapshotsIT`

Reply:
Done!